docs/src/concepts.md

```diff
@@ -0,0 +1,116 @@
+
+## Key Partnerships
+
+Over the past few years, several labs have developed DataJoint-based data management and processing pipelines for Neuropixels probes. Our team collaborated with several of them during their projects. Additionally, we interviewed these teams to understand their experimental workflow, pipeline design, associated tools, and interfaces. These teams include:
+
+- [International Brain Lab](https://internationalbrainlab.org) - [https://github.com/int-brain-lab/IBL-pipeline](https://github.com/int-brain-lab/IBL-pipeline)
+
+- [Mesoscale Activity Project (HHMI Janelia)](https://github.com/mesoscale-activity-map) - [https://github.com/mesoscale-activity-map/map-ephys](https://github.com/mesoscale-activity-map/map-ephys)
+
+- Moser Group (Norwegian University of Science and Technology) - see [pipeline design](https://moser-pipelines.readthedocs.io/en/latest/ephys/overview.html)
+
+- Andreas Tolias Lab (Baylor College of Medicine)
+
+- BrainCoGs (Princeton Neuroscience Institute)
+
+- Brody Lab (Princeton University)
+
+## Element Architecture
+
+Each of the DataJoint Elements are a set of tables for common neuroinformatics modalities to organize, preprocess, and analyze data. Each node in the following diagram is either a table in the Element itself or a table that would be connected to the Element.
+
+![element-array-ephys diagram](https://raw.githubusercontent.com/datajoint/element-array-ephys/main/images/attached_array_ephys_element_acute.svg)
+
+### `subject` schema ([API docs](https://datajoint.com/docs/elements/element-animal/api/element_animal/subject))
+- Although not required, most choose to connect the `Session` table to a `Subject` table.
+
+| Table | Description |
+| --- | --- |
+| Subject | Basic information of the research subject |
+
+### `session` schema ([API docs](https://datajoint.com/docs/elements/element-session/api/element_session/session_with_datetime))
+
+| Table | Description |
+| --- | --- |
+| Session | Unique experimental session identifier |
+
+### `probe` schema ([API docs](../api/element_array_ephys/probe))
+Tables related to the Neuropixels probe and electrode configuration.
+
+| Table | Description |
+| --- | --- |
+| ProbeType | a lookup table specifying the type of Neuropixels probe (e.g. "neuropixels 1.0", "neuropixels 2.0 single-shank"). |
```

Comment:
Please follow the convention above, with the description in sentence case

Reply:
Done